lib/GNOS/SampleInformation.pm

```diff
@@ -98,12 +101,12 @@ sub get {
         my $attempts = 0;
 
         while ($status == 0 and $attempts < 10) {
-            $status = $self->download_analysis($analysis_full_url, $analysis_xml_path, $use_cached_xml);
+            $status = $self->download_analysis($analysis_full_url, $analysis_xml_path, $use_cached_analysis);
             $attempts++;
         }
 
         if (not -e $analysis_xml_path or not eval {$xs->XMLin($analysis_xml_path); } ) {
-           say $parse_log "skipping $analysis_id - no xml file available: $analysis_xml_path";
+           say $parse_log "SKIPPING: $analysis_id - no xml file available: $analysis_xml_path";
```

Comment:
I hav echanged it for a few that are actually dieing to say so! ... Includign this one.